src/pymedphys/_labs/paulking/profile.py

```diff
@@ -24,27 +24,56 @@
 # program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
 
 
+from copy import deepcopy
 from typing import Callable
 
 import numpy as np
 from scipy import interpolate
 import matplotlib.pyplot as plt
 
+import xarray as xr
+
 from ...libutils import get_imports
-from ...xarray import XArrayComposition
+# from .._level1.coreobjects import _PyMedPhysBase
 IMPORTS = get_imports(globals())
 
 
 NumpyFunction = Callable[[np.ndarray], np.ndarray]
 
-
+#
 # pylint: disable = C0103, C0121
 
 
-class Dose1D(XArrayComposition):
```

Comment:
Was removing the inheritance here on purpose?

Reply:
No. On my current version it appears as follows:

class Dose1D(DoseBase):

Had some difficulties with the move to GitHub, adding a branch, and re-organizing files. In this file, made no intentional changes above the DoseProfile class, and only additions, there.

Reply:
No.

Reply:
All good. Can sort that out later.